functions/notify_slack_test.py

```diff
@@ -1,204 +1,121 @@
-#!/usr/bin/env pytest
+# -*- coding: utf-8 -*-
+"""
+    Slack Notification Test
+    -----------------------
+
+    Unit tests for `notify_slack.py`
+
+"""
+
+import ast
+import os
 
 import notify_slack
 import pytest
-from json import loads
-from os import environ
-
-events = (
-    (
-        {
```

Comment:
this threw me for a loop. some events in the SNS event format thats delivered to the lambda, and others in the raw message format (that would in theory be passed to something like `aws sns publish --topic-arn "xxx" --message "<message>"`)